pyproject.toml

```diff
@@ -75,16 +75,11 @@ protocols = ["qadence-protocols"]
 libs = ["qadence-libs"]
 dlprof = ["nvidia-pyindex", "nvidia-dlprof[pytorch]"]
 all = [
-  "pulser-core==0.18.1",
-  "pulser-simulation==0.18.1",
-  "pasqal-cloud==0.8.1",
-  "amazon-braket-sdk<1.71.2",
```

Comment:
Maybe we can add braket to the test env ?

Reply:
I just refactored what was there, it's the same as before. The test env uses `all` which still has `braket` on it.